eng/release/Scripts/PublishToBlobAccounts.ps1

```diff
@@ -26,7 +26,7 @@ function Generate-Source-Uri{
         [Parameter(Mandatory=$true)][string]$AssetType
     )
 
-    return "https://$sourceAccountName.blob.core.windows.net/$sourceContainerName/$BuildNumber/${AssetType}Assets/*"
```

Comment:
Will the docker side also need to be updated for this?

Reply:
Yes, dotnet-docker will need to be updated. It currently doesn't know how to get .NET Monitor from the staging account, so these changes don't really impact that ability, but I will be fixing the updater to understand the layout.